### PR TITLE
Jedwards/mergetob896c79

### PR DESCRIPTION
**Pull request by jedwards4b**
Merge to cime master
Test suite: scripts_regression_tests.py
Test baseline: 
Test namelist changes: 
Test status: roundoff - mapping files and some stream files were updated

Fixes 

User interface changes?: 

Update gh-pages html (Y/N)?:

Code review:

### CODE REVIEW

**Comment by uturuncoglu** [comment]
Here the list of failed tests

./cs.status.20190502_145112_e2dgee | grep FAIL
  ERR_Vnuopc_Ld5.f19_g16.BMOM.cheyenne_intel.allactive-nuopc_cap_io (Overall: FAIL) details:
    FAIL ERR_Vnuopc_Ld5.f19_g16.BMOM.cheyenne_intel.allactive-nuopc_cap_io COMPARE_base_rest
    FAIL ERR_Vnuopc_Ld5.f19_g16.BMOM.cheyenne_intel.allactive-nuopc_cap_io BASELINE may1intel19: DIFF
  ERS_Vnuopc_Ld5.T62_g16.CMOM.cheyenne_intel (Overall: FAIL) details:
    FAIL ERS_Vnuopc_Ld5.T62_g16.CMOM.cheyenne_intel RUN time=259
  ERS_Vnuopc_Ld5.T62_g16.GMOM.cheyenne_intel (Overall: FAIL) details:
    FAIL ERS_Vnuopc_Ld5.T62_g16.GMOM.cheyenne_intel COMPARE_base_rest
    FAIL ERS_Vnuopc_Ln5.f19_g17.F2000Nuopc.cheyenne_intel.cam-nuopc_cap BASELINE may1intel19: DIFF
    FAIL ERS_Vnuopc_Ln5.f45_f45_mg37.I2000Clm50SpNuopc.cheyenne_intel.clm-nuopc_cap NLCOMP
    FAIL ERS_Vnuopc_Ln5.f45_f45_mg37.I2000Clm50SpNuopc.cheyenne_intel.clm-nuopc_cap BASELINE may1intel19: DIFF
    FAIL ERS_Vnuopc_Ln5.ne16_ne16_mg17.QPC4.cheyenne_intel.cam-nuopc_cap TPUTCOMP Error: Computation time increase > 25 pct from baseline
    FAIL ERS_Vnuopc_Ln9.f19_g17.X.cheyenne_intel BASELINE may1intel19: DIFF
    FAIL ERS_Vnuopc_Ln9_N3.f19_g17_rx1.A.cheyenne_intel BASELINE may1intel19: DIFF
    FAIL ERS_Vnuopc_Ln9_N3.f19_g17_rx1.A.cheyenne_intel TPUTCOMP Error: Computation time increase > 25 pct from baseline
    FAIL SMS_Vnuopc.f19_g17.X.cheyenne_intel BASELINE may1intel19: DIFF
    FAIL SMS_Vnuopc_Ld1_N3.f19_g17_rx1.A.cheyenne_intel BASELINE may1intel19: DIFF
    FAIL SMS_Vnuopc_Ld1_N3.f19_g17_rx1.A.cheyenne_intel TPUTCOMP Error: Computation time increase > 25 pct from baseline
    FAIL SMS_Vnuopc_Ld5.f19_g16.BMOM.cheyenne_intel.allactive-nuopc_cap_io BASELINE may1intel19: DIFF
    FAIL SMS_Vnuopc_Ld5.f19_g16.BMOM.cheyenne_intel.allactive-nuopc_cap_io MEMCOMP Error: Memory usage increase > 10% from baseline
    FAIL SMS_Vnuopc_Ln5.f19_f19_mg17.I2000Clm50SpGs.cheyenne_intel.clm-nuopc_cap NLCOMP
    FAIL SMS_Vnuopc_Ln5.f19_f19_mg17.I2000Clm50SpGs.cheyenne_intel.clm-nuopc_cap BASELINE may1intel19: DIFF
    FAIL SMS_Vnuopc_Ln5.f19_g17.F2000Nuopc.cheyenne_intel.cam-nuopc_cap BASELINE may1intel19: DIFF
    FAIL SMS_Vnuopc_Ln5.f45_f45_mg37.I2000Clm50SpNuopc.cheyenne_intel.clm-nuopc_cap NLCOMP
    FAIL SMS_Vnuopc_Ln5.f45_f45_mg37.I2000Clm50SpNuopc.cheyenne_intel.clm-nuopc_cap BASELINE may1intel19: DIFF

File: config/cesm/config_grids.xml [hunk 1]
```diff
@@ -1380,8 +1398,8 @@
 
     <domain name="ne0np4CONUS.ne30x8">
       <nx>174098</nx> <ny>1</ny>
-      <file grid="atm|lnd" mask="gx1v7">$DIN_LOC_ROOT/share/domains/domain.lnd.ne0CONUSne30x8_gx1v7.190304.nc</file>
-      <file grid="ocnice"  mask="gx1v7">$DIN_LOC_ROOT/share/domains/domain.ocn.ne0CONUSne30x8_gx1v7.190304.nc</file>
+      <file grid="atm|lnd" mask="gx1v7">$DIN_LOC_ROOT/share/domains/domain.lnd.ne0CONUSne30x8_gx1v7.190322.nc</file>
```
Comment (jedwards4b) [inline config/cesm/config_grids.xml:1401]: I think that these grid map changes explain the differences from baselines.

**Comment by jedwards4b** [comment]
I believe that differences from baselines can be explained by changes in config/cesm/config_grids_*.xml
you can rerun with the old settings to confirm.   The run fail was a cheyenne system problem, you should try resubmitting that job.  Stream inputs were also changes for I compsets which may be an additional answer change.

File: config/cesm/config_grids.xml (hunk 1 above)
Comment (uturuncoglu) [reply config/cesm/config_grids.xml:1401]: testing again with old files to see what happens.

**Comment by uturuncoglu** [comment]
I performed new test with old XML files. The list of failed tests can be seen as follows,

./cs.status.20190503_095900_fad03o | grep FAIL
  ERR_Vnuopc_Ld5.f19_g16.BMOM.cheyenne_intel.allactive-nuopc_cap_io (Overall: FAIL) details:
    FAIL ERR_Vnuopc_Ld5.f19_g16.BMOM.cheyenne_intel.allactive-nuopc_cap_io COMPARE_base_rest
    FAIL ERR_Vnuopc_Ld5.f19_g16.BMOM.cheyenne_intel.allactive-nuopc_cap_io BASELINE may1intel19: DIFF
  ERS_Vnuopc_Ld5.T62_g16.CMOM.cheyenne_intel (Overall: FAIL) details:
    FAIL ERS_Vnuopc_Ld5.T62_g16.CMOM.cheyenne_intel COMPARE_base_rest
  ERS_Vnuopc_Ld5.T62_g16.GMOM.cheyenne_intel (Overall: FAIL) details:
    FAIL ERS_Vnuopc_Ld5.T62_g16.GMOM.cheyenne_intel COMPARE_base_rest
    FAIL ERS_Vnuopc_Ln5.f19_g17.F2000Nuopc.cheyenne_intel.cam-nuopc_cap BASELINE may1intel19: DIFF
    FAIL ERS_Vnuopc_Ln5.f45_f45_mg37.I2000Clm50SpNuopc.cheyenne_intel.clm-nuopc_cap NLCOMP
    FAIL ERS_Vnuopc_Ln5.f45_f45_mg37.I2000Clm50SpNuopc.cheyenne_intel.clm-nuopc_cap BASELINE may1intel19: DIFF
    FAIL ERS_Vnuopc_Ln9.f19_g17.X.cheyenne_intel BASELINE may1intel19: DIFF
    FAIL ERS_Vnuopc_Ln9_N3.f19_g17_rx1.A.cheyenne_intel BASELINE may1intel19: DIFF
    FAIL ERS_Vnuopc_Ln9_N3.f19_g17_rx1.A.cheyenne_intel TPUTCOMP Error: Computation time increase > 25 pct from baseline
    FAIL SMS_Vnuopc.f19_g17.X.cheyenne_intel BASELINE may1intel19: DIFF
    FAIL SMS_Vnuopc_Ld1_N3.f19_g17_rx1.A.cheyenne_intel BASELINE may1intel19: DIFF
    FAIL SMS_Vnuopc_Ld1_N3.f19_g17_rx1.A.cheyenne_intel TPUTCOMP Error: Computation time increase > 25 pct from baseline
    FAIL SMS_Vnuopc_Ld5.f19_g16.BMOM.cheyenne_intel.allactive-nuopc_cap_io BASELINE may1intel19: DIFF
    FAIL SMS_Vnuopc_Ld5.f19_g16.BMOM.cheyenne_intel.allactive-nuopc_cap_io MEMCOMP Error: Memory usage increase > 10% from baseline
    FAIL SMS_Vnuopc_Ld5.T62_g37.DTEST.cheyenne_intel.cice-nuopc_cap MEMCOMP Error: Memory usage increase > 10% from baseline
    FAIL SMS_Vnuopc_Ln5.f19_f19_mg17.I2000Clm50SpGs.cheyenne_intel.clm-nuopc_cap NLCOMP
    FAIL SMS_Vnuopc_Ln5.f19_f19_mg17.I2000Clm50SpGs.cheyenne_intel.clm-nuopc_cap BASELINE may1intel19: DIFF
    FAIL SMS_Vnuopc_Ln5.f19_g17.F2000Nuopc.cheyenne_intel.cam-nuopc_cap BASELINE may1intel19: DIFF
    FAIL SMS_Vnuopc_Ln5.f45_f45_mg37.I2000Clm50SpNuopc.cheyenne_intel.clm-nuopc_cap NLCOMP
    FAIL SMS_Vnuopc_Ln5.f45_f45_mg37.I2000Clm50SpNuopc.cheyenne_intel.clm-nuopc_cap BASELINE may1intel19: DIFF

**Comment by uturuncoglu** [comment]
I tested cmeps-cime without merging with PR #89 and it has only three tests that fail

  ERR_Vnuopc_Ld5.f19_g16.BMOM.cheyenne_intel.allactive-nuopc_cap_io (Overall: FAIL) details:
    FAIL ERR_Vnuopc_Ld5.f19_g16.BMOM.cheyenne_intel.allactive-nuopc_cap_io COMPARE_base_rest
  ERS_Vnuopc_Ld5.T62_g16.CMOM.cheyenne_intel (Overall: FAIL) details:
    FAIL ERS_Vnuopc_Ld5.T62_g16.CMOM.cheyenne_intel COMPARE_base_rest
  ERS_Vnuopc_Ld5.T62_g16.GMOM.cheyenne_intel (Overall: FAIL) details:
    FAIL ERS_Vnuopc_Ld5.T62_g16.GMOM.cheyenne_intel COMPARE_base_rest

So, I think that those tests will also fails after merge but rest of them needs to be investigated.

**Pull request merged**